.github/workflows/release.yml

```diff
@@ -0,0 +1,94 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        description: "Prerelease"
+        required: true
+        default: false
+        type: boolean
+      draft:
+        description: "Draft"
+        required: true
+        default: false
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    name: Version Bump and Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AUTOMATION_USER_TOKEN }}
+          fetch-depth: 0
+
+      - name: Get latest non-prerelease release
+        id: latestrelease
+        uses: cardinalby/git-get-release-action@v1
+        with:
+          latest: true
+          prerelease: false
+          draft: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get next version
+        id: semverbump
+        uses: WyriHaximus/github-action-next-semvers@v1.2.1
+        with:
+          version: ${{ steps.latestrelease.outputs.tag_name }}
+
+      - name: Update Version in code
+        run: |
+          sed -i "s/VERSION = '[0-9].[0-9].[0-9]'/VERSION = '${{steps.semverbump.outputs.minor}}'/g" ./lib/devcycle-ruby-server-sdk/version.rb
```

Comment:
Won't this fail to match multi-digit numbers in the version string?
e.g. `1.12.4` is a valid version that we already publish for the js SDK.

I think some of the other release workflows might have this bug as well.

Reply:
shit, you are totally right

Reply:
ok, i think i fixed it: `[0-9]+\.[0-9]+\.[0-9]+`

Reply:
I might be wrong here, but I think sed needs `\+` instead of `+`?
At least it did for my local gnu sed when I tried it out.